libs/openchallenges/challenge-search/src/lib/challenge-search.component.scss

```diff
@@ -43,3 +45,12 @@
   padding-top: 0;
   padding-bottom: 0;
 }
+::ng-deep .p-multiselect.p-component {
```

Comment:
Something for us to keep in mind is that `ng-deep` will be deprecated one day.  But I think we can keep using it until W3C has an official replacement.